frontend/src/containers/AddTable.tsx

```diff
@@ -165,6 +165,7 @@ function AddTable() {
       dynamicTyping: true,
       skipEmptyLines: true,
       comments: "#",
+      encoding: "ISO-8859-1",
```

Comment:
The fact that you had to make the change in 4 different files speaks about our opportunity to DRY-out the parsing. If you have a chance later this sprint, could you please do the refactor of extracting this parsing function to it's own `CsvParserService`?